modules/kueue-release-notes-1.2.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="release-notes-1.2_{context}"]
+= Release notes for {kueue-name} version 1.2
+
+[role="_abstract"]
+{kueue-name} version 1.2 is a generally available release that is supported on {product-title} versions 4.18 and later. {kueue-name} version 1.2 uses link:https://kueue.sigs.k8s.io/docs/overview/[Kueue] version 0.14.
+
+[id="release-notes-1.2-new-features_{context}"]
+== New features and enhancements
+
+Monitoring of pending workloads::
+{kueue-name} version 1.2 provides the `VisibilityOnDemand` feature to monitor the pipeline of pending jobs in the cluster queue and the local queue, and help users to estimate when their jobs will start. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.21/html/ai_workloads/red-hat-build-of-kueue#monitoring-pending-workloads.html[Monitoring pending workloads].
+
+[id="release-notes-1.2-fixed-issues_{context}"]
+== Fixed issues
+
+Custom resources are not deleted properly when you uninstall {kueue-name}::
+After you uninstall the {kueue-op} using the *Delete all operand instances for this operator* option in the {product-title} web console, {kueue-name} custom resources were attempted to be deleted. With this release, they are not considered for deletion.
++
+(link:https://issues.redhat.com/browse/OCPBUGS-62254[OCPBUGS-62254])
+
+Documentation error in previous versions of {kueue-name}::
+In link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/ai_workloads/red-hat-build-of-kueue#create-kueue-cr_install-kueue[Creating a Kueue custom resource], the optional workload types `Pod`, `Deployment`, `StatefulSet` were omitted. They are now included. 
++
+(link:https://issues.redhat.com/browse/OCPBUGS-62877[OCPBUGS-62877])
+
+{kueue-name} metrics were not being exposed to Prometheus from version 1.1::
+Prometheus was not scraping metrics from the operator's controller, even though the ServiceMonitor and RBAC resources were successfully created as part of the operator installation. As a result, none of the Kueue metrics were available in the cluster monitoring stack. 
++
+The metrics-service added during the installation was configured with an incorrect port reference, causing Prometheus to fail in scraping metrics from the Kueue endpoint. The port name has been updated with the correct port name.
```

Comment:
Seems like this should be two words unless it's a literal item that should be in backticks :thinking: 
```suggestion
The metrics service added during the installation was configured with an incorrect port reference, causing Prometheus to fail in scraping metrics from the Kueue endpoint. The port name has been updated with the correct port name.
```

Reply:
Made this change.